Strategies/protocols/psyfi/actions/deposit.ts

```diff
@@ -101,11 +153,17 @@ export const deposit = async (
       transferAddress,
```

Comment:
this should be correct, if this is the token account to transfer from on the UI (unless its for SOL deposits that goes through the wSOL ATA)

Reply:
the owner and transferAddress are same on the UI so that's why was a bit confused!